bot/exts/help_channels/_cog.py

```diff
@@ -507,7 +507,7 @@ async def wait_for_dormant_channel(self) -> discord.TextChannel:
         """Wait for a dormant channel to become available in the queue and return it."""
         log.trace("Waiting for a dormant channel.")
 
-        task = asyncio.create_task(self.channel_queue.get())
+        task = scheduling.create_task(self.channel_queue.get())
```

Comment:
Any reason not to use the bot loop here?

Reply:
If the loop is not passed in, the util itself uses asyncio.create_task which gets the running loop